whisper_online.py

```diff
@@ -645,16 +660,16 @@ def output_transcript(o, now=None):
             print("%1.4f %1.0f %1.0f %s" % (now*1000, o[0]*1000,o[1]*1000,o[2]),file=logfile,flush=True)
             print("%1.4f %1.0f %1.0f %s" % (now*1000, o[0]*1000,o[1]*1000,o[2]),flush=True)
         else:
-            print(o,file=logfile,flush=True)
+            # No text, so no output
+            pass
 
     if args.offline: ## offline mode processing (for testing/debugging)
         a = load_audio(audio_path)
         online.insert_audio_chunk(a)
         try:
             o = online.process_iter()
-        except AssertionError:
-            print("assertion error",file=logfile)
-            pass
+        except AssertionError as e:
+            log.error(f"assertion error: {repr(e)}")
```

Comment:
I have never seen these errors, would a full backtrace be useful here? (If so, `log.exception` would be better.)

Reply:
If we were making changes here I'd be inclined to go the other way and say that a failed assertion should just crash the process.  `assert` should be telling us that something's coded wrong, so to a certain degree all bets are off.  That being said, given that I don't know what the original problem was that caused these exception handlers to be written, `logging.exception` is a good compromise if that seems overly harsh.